README.md

```diff
@@ -2,11 +2,20 @@
 
 ⚠️ **Warning: this project is still very alpha**
 
-This project is:
+React component variations are:
 * A format called "variations" for specifying React component **_examples_** and providing **_documentation_** for them.
 * Tools for finding, validating, and iterating over variations.
+* A collection of "consumers" for doing useful things with variations.
 
-A variation looks like:
+## Table of contents
+
+- [Variations](#variations)
+- [Consumers](#consumers)
+- [Contribution Guidelines](#contribution-guidelines)
+
+## Variations
```

Comment:
Does this work by default with the table of contents, or does it need to have an explicit anchor link?

```md
[Variations](#variations)

## <a name="variations"></a>Variations
```

Reply:
It totally works